examples/complete/main.tf

```diff
@@ -70,6 +70,38 @@ module "create_scc_instance" {
   wp_instance_crn                   = module.scc_wp.crn
 }
 
+##############################################################################
+# SCC custom profile
+##############################################################################
+
+module "create_scc_profile" {
+  source      = "../../modules/profile/."
+  instance_id = module.create_scc_instance.guid
+  controls = [
+    {
+      control_library_name    = "IBM Cloud Framework for Financial Services",
+      control_library_version = "1.6.0"
+      control_list = [
+        "AC",
+        "AC-1",
+        "AC-1(a)",
+      ]
+    },
+    {
+      control_library_name    = "CIS IBM Cloud Foundations Benchmark",
+      control_library_version = "1.0.0"
+      control_list = [
+        "1.16",
+        "1.18",
+        "1.19",
+        "1.4",
+      ]
+    },
+  ]
```

Comment:
same as usge comment - add another control library example here which does not specify a control_list (in which case it adds all control right?)
Also add some code comments here to explain